### DISCUSSION TITLE
Problem with xunit.analyzers 0.10.0 when installing via upstream feed

### DISCUSSION THREAD

**Discussion by janban-outlook** [xunit/xunit#2256] [Question] [answered]
Hi,

Following the latest recommendations we no longer use the public library next to our private nuget feed in Azure Devops.
So we only use our private feed, and added nuget.org as an upstream package link.
This works fine for everything (everything we need, that is), except for the most recent xunit.analyzers package.

>install-package xunit.analyzers -version 0.10.0 says it cannot be found
>install-package xunit.analyzers -version 0.9.0 works

I cannot find any explanation for this. Does anybody know what's happening?

Thanks for any hint,
Jan van Veldhuizen

**Comment by bradwilson** [comment]
I do not. I would suggest enlisting the help of either the NuGet team or the owners of your upstream package feed in Azure DevOps.

**Comment by janban-outlook** [comment, marked as answer]
This discussion can be closed.
I was looking in the wrong direction. The problem was not related to xunit at all.

Azure Devops nuget feeds appear to be immutable, and I was unaware of the fact that version 0.10.0 was removed from the feed a while ago. This explains the behavior.
